lisa/util/constants.py

```diff
@@ -46,8 +46,8 @@
 
 ENVIRONMENT = "environment"
 ENVIRONMENTS = "environments"
-ENVIRONMENTS_TEMPLATE = "template"
-ENVIRONMENTS_NODES_SPEC = "spec"
+ENVIRONMENTS_REQUIREMENT = "requirements"
+ENVIRONMENTS_NODES_REQUIREMENT = "requirement"
```

Comment:
This confuses me since the difference is just the plurality of the word. I can't from reading this determine what these constants may be used for.

Reply:
oops. The design was changed, removed the requirements.